trunk/modules/python/dataset/ingest/labeled.py

```diff
@@ -1,6 +1,13 @@
 import os
 import numpy as np
 
+def mostCommon(stuff, func):
+    '''Find the most common result of func yielded by stuff'''
+    from collections import Counter
+    counter = Counter([func(s) for s in stuff])
+    common = counter.most_common()[0][0] # first tuple, first elem
+    return common

```

Comment:
just return the result of line 8 directly. Also comment should be added on the line before.